appointments.py

```diff
@@ -125,12 +125,37 @@ async def on_connect(websocket, path):
     finally:
         connected_clients.remove(websocket)
 
+#  Telegram Notifcation Added by Siva Rajendran
+
+def telegram_notify(message):
+    with open('./config/config.json', 'r') as config_file:
```

Comment:
We need to guard against non-existing config files/invalid config and only continue when `notify` Bot created